examples/ethereum-tracker/src/contract.rs

```diff
@@ -43,14 +43,14 @@ impl Contract for EthereumTrackerContract {
         // Validate that the application parameters were configured correctly.
         self.runtime.application_parameters();
         self.state.argument.set(argument);
```

Comment:
Do we need to keep the original start block?

Reply:
No, we don't. That is an optimization that can be done indeed. Let me go at it.

Reply:
Done.